.gitignore

```diff
@@ -2,6 +2,7 @@
 tmp
 .DS_Store
 docker-compose.override.yml
+docker-compose.dev.yml
```

Comment:
i added a local docker override to run pgadmin in locally to mess around with the database GUI :)

Reply:
yeah that's good, I'd added the line 4 earlier, my override is `docker-compose.override.yml` previously. How about we make the file `docker-compose.override.dev.yml` or something so that it's clear that it's an override file that's for local dev?